src/patternfly/components/Form/form.scss

```diff
@@ -167,6 +168,10 @@
     color: var(--pf-c-form--m-error--Color);
   }
 
+  &.pf-m-validated {
```

Comment:
I wonder if we should remove the `:not(.pf-m-error)` selector, and just apply `color: var(--pf-c-form__helper-text--Color);` to `.pf-c-form__helper-text`, then overwrite `--pf-c-form__helper-text--Color` in `.pf-m-error` and `.pf-m-validated`. Can you try that and see if it works?

Reply:
seems to work fine as it should. Not sure why the `:not(.pf-m-error)` and then `&.pf-m-error` was there to begin with.